multi_tenant_test.go

```diff
@@ -314,6 +314,13 @@ func testMultiTenantByAppType(t *testing.T, appType string) {
 			).Scan(&count)
 			require.NoError(t, err)
 			require.Equal(t, 1, count)
+
+			var userId string
```

Comment:
tests should be single purpose too :)
You can imagine what can happen as we are adding more and more scenarios in a single test

Reply:
Yeah I am adding integration tests under the gateway folder. Will clean this up from here